ios/TelevisionAcademyIOSPlayerPlugin/PluginClasses/PlayerViewController.swift

```diff
@@ -99,29 +110,73 @@ extension PlayerViewController: UserInterfaceListener {
 extension PlayerViewController: PlayerListener {
 
     func onReady(_ event: ReadyEvent) {
+
         guard let item = self.player.config.sourceItem as? PlayableSourceItem,
             let elapsedTime = item.elapsedTime else {
                 return
         }
 
         let time = elapsedTime/Double(Constants.miliseconds.rawValue)
-        self.player.seek(time: time)
+        player.seek(time: time)
```

Comment:
If elapsedTime is not defined, we will not have analytics initialisation

Reply:
Hi @VladSumtsov , the issue is fixed.